concepts/reduce/about.md

```diff
@@ -0,0 +1,93 @@
+# About
+
+Suppose we want to sum an array of numbers.
+There are many ways to accomplish this goal.
+
+- In many languages, this problem is expressed as a loop with an accumulator variable.
+- This problem can be written as a recursive function.
+  In pseudo-code, we might have this.
+
+  ```none
```

Comment:
Completely as an aside, this `none` got me wondering if there was some sort of pseudocode standards/"language" for syntax highlighting. That led me to [this standard](https://users.csc.calpoly.edu/~jdalbey/SWE/pdl_std.html) and [this pseudocode language](https://en.wikiversity.org/wiki/Pseudocode/Gaddis_Pseudocode) (is that an oxymoron?). I'm not sure if Wikipedia has some sort of standard or whatnot for pseudocode highlighting.